src/icons/imported/Icon/BoxLogo.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+/* eslint-disable react/jsx-sort-props */
+import * as React from 'react';
+import AccessibleSVG from '../../accessible-svg';
+
+type SvgProps = {
```

Comment:
Can this type be shared between the components?

Reply:
Possibly, although we haven't done that yet for many of the other icons. I was considering introducing a type alongside `AccessibleSVG` and having the importer reference that type. Something like that?

Reply:
I think @tjgupta just did [something similar](https://github.com/box/box-ui-elements/pull/1722/files#diff-0289c5bf2a1b179fed23b0670fc2bcaeR7) for our general icons.

Reply:
Oh this is *very* new. Thanks for pointing out.

Reply:
actually, looks like I added the `Icon` flow type earlier this year, but it wasn't used across the board yet. I remember Tim and I working thru all that back then.

I'll update this to import that type instead of in-lining them, which should be fine.